### ISSUE TITLE
Support 80, 128, 160, 256 and 512 bit registers in the RReg

### ISSUE THREAD

**Issue by XVilka** [radare/radare2#3130]
For the glory of modern processors.

**Comment by condret** [comment]
buäh, that means using bignum, so much pain in esil

**Comment by radare** [comment]
I have implemented support for long double (80 bit) and packed registers. so all this stuff can be implemented just by reading the packs of data inside. This needs a bit more testing with real backends. But in theory, rreg now supports all what we need to acomplish this. I have also added the `drm` command to read and write packed register values, which can be also used with GPR registers like this:
```
drm rip 0 32 # get lower 32bits of the RIP address
drm rip 1 32 # get higher 32bit ...
```

I think that handling those values as packed data is the way to go. But at some point we can extend the API to support bignum. in fact, there's already an API for that in `util/big.c`. But I dont think we want to go that way yet. Most >64 bit registers are accessed as packed registers, which are smaller and used for paralel operations like MMX and such.

Which arches have >64 bit integer registers?

Feel free to close when you test it and get happy about it.
